packages/client/tests/e2e/init-example-env/src/index.ts

```diff
@@ -0,0 +1,12 @@
+import { readFileSync } from 'node:fs'
+
+const exampleEnvFiles = ['.env.example', '.example.env']
+
+exampleEnvFiles.forEach((file) => {
+  const fileContents = readFileSync(file, 'utf8')
+
+  if (!fileContents.includes('DATABASE_URL="file:./dev.db"')) {
+    console.error(`${file} does not contain env var from init command"`)
+    process.exit(1)
+  }
```

Comment:
This doesn't actually test that the example file was used. You probably need to assert that `FOO=BAR` is present.